docs/en/pact-functions.md

```diff
@@ -1370,6 +1382,18 @@ Continue previously-initiated pact identified by PACT-ID at STEP, optionally spe
 ```
 
 
+### env-chain-data {#env-chain-data}
+
+*new-data*&nbsp;`object:[]*` *&rarr;*&nbsp;`string`
+
+
+Update existing entries 'chain-data' with NEW-DATA, replacing those items only.
+```lisp
+pact> (env-chain-data { "chain-id": 2, "block-height": 20 })
+"Updated public metadata"
+```
+
+
```

Comment:
Ah thank you - I realized i forgot to do a complete build locally on my machine to produce the docs